bigchaindb/core.py

```diff
@@ -219,17 +221,26 @@ def get_owned_ids(self, owner):
             list: list of `txids` currently owned by `owner`
         """
 
-        response = r.table('bigchain')\
-                    .concat_map(lambda doc: doc['block']['transactions'])\
-                    .filter({'transaction': {'new_owner': owner}})\
-                    .pluck('id')['id']\
-                    .run(self.conn)
+        response = r.table('bigchain') \
+            .concat_map(lambda doc: doc['block']['transactions']) \
```

Comment:
Not according to the documentation. Seems like r.row does not work for all cases.
> Note that row does not work within subqueries to access nested documents; you should use anonymous functions to access those documents instead